code/modules/admin/admin.dm

```diff
@@ -739,7 +739,6 @@ var/global/floorIsLava = 0
 			<A href='?src=\ref[src];secretsfun=meaty_gores'>Trigger an Organic Debris Field</A><BR>
 			<A href='?src=\ref[src];secretsfun=lightsout'>Toggle a "lights out" event</A><BR>
 			<A href='?src=\ref[src];secretsfun=mass_hallucination'>Cause the crew to hallucinate</A><BR>
-			<A href='?src=\ref[src];secretsfun=mass_drunk'>Cause the crew to become drunk</A><BR>
```

Comment:
If it were to come back, it'd probably have to be coded by someone other than me to add a lot more contingencies and whatnot. I just threw it in based on a suggestion in the thread when I was making the random event update.